bin/crew

```diff
@@ -9,6 +9,7 @@ require 'digest/sha2'
 require 'json'
 require 'fileutils'
 require 'tmpdir'
+require 'resolv-replace'
```

Comment:
Maybe move this line to `lib/downloader.rb`? (since it is not necessary for `crew` itself)

Reply:
done